scripts/examples-runner.ts

```diff
@@ -0,0 +1,23 @@
+import {extname, join} from "path";
+import {readdir} from "fs";
+
+const pkg = process.argv[3]
+if (!pkg) {
+  throw new Error("Package argument must be passed")
+}
+
+const examplesPath = `../packages/${pkg}/examples/`
+const path = join(__dirname, examplesPath)
+
+readdir(path, async (err, files) => {
+  if (err) {
+    return console.log("Error loading examples directory: ", err)
+  }
+  const importedFiles = files.map(fileName => {
+    if (extname(fileName) === ".ts") {
+      return import(examplesPath + fileName);
+    }
+  }).filter(file => file)
+
+  await Promise.all(importedFiles)
+})
```

Comment:
Do I get this right that this is only for CI and one can still run the examples "stand-alone"?

Reply:
That's correct! Actually, this is a workaround because I thought we could just use `ts-node examples/*.ts` and it would run all the ts files, but it doesn't :(